package.json

```diff
@@ -23,8 +23,8 @@
     "lint": "eslint src scripts",
     "build:js": "webpack --mode production",
     "build:js-dev": "webpack --mode development",
-    "build:py": "node ./extract-meta src/components > dash_html_components/metadata.json && cp package.json dash_html_components && dash-generate-components ./src/components dash_html_components",
-    "build:r": "node ./extract-meta src/components > dash_html_components/metadata.json && cp package.json dash_html_components && dash-generate-components ./src/components dash_html_components --r-prefix 'html'",
+    "build:py": "dash-generate-components ./src/components dash_html_components",    
+    "build:r": "dash-generate-components ./src/components dash_html_components --r-prefix 'html'",
```

Comment:
Ha good call, `dash-generate-components` does all of this and has for quite some time!

Reply:
In fact now we can 🔪 `extract-meta` from this repo, right?

Reply:
:beers:

Reply:
fixed in f3f33856d95be679db1ceb931502b0ea819712c4